### DISCUSSION TITLE
[Data-Importer] Develop image broke authentication

### DISCUSSION THREAD

**Discussion by MacPaille** [firefly-iii/firefly-iii#5490] [General]
Hey @JC5 
Seems the latest develop image broke the authentication to firefly. I'm running both latest (build 53) image and develop (build 56) in containers, both using the same token to auth and develop does not work. Getting this error:

[2022-01-05 18:05:48] production.ERROR: Could not connect to Firefly III: Error accessing "https://firefly.myfakedomain.com/api/v1/about". Status code is 401. Body is: {"message":"Unauthenticated","exception":"AuthenticationException"}

**Comment by JC5** [comment]
I'm not seeing any problems myself. There haven't been any relevant changes since 5.6.9.

**Reply by MacPaille** [reply]
Strange. I'm running FF in 5.6.9 and even if I click on reauthenticate in the importer the develop image fails with that 401 where the latest image works fine

**Reply by MacPaille** [reply]
OK it seems to work now I create a new token. So each image has it's own token to auth to firefly :)

**Reply by animalillo** [reply]
I have recently made a fresh firefly install and i don't seem to manage to authenticate against the api, neither using the importer nor doing http requests to the api (using version 5.6.10 of Firefly III installed via composer).

I also tried using the oauth method on the importer, but no luck with that either.

**Reply by JC5** [reply]
Thanks for letting me know. Please open your own ticket if you need help.
